### DISCUSSION TITLE
Installation fails in a Webpack build

### DISCUSSION THREAD

**Discussion by TheDirigible** [inkline/inkline#310] [Need help]
Installation with Vue Cli worked fine, but with a webpack based build (vue as a library), I get 175 errors.
This webpack configuration works for 10 other component libs.

ERROR in ./node_modules/@inkline/inkline/components/IBadge/script.mjs?vue&type=script&lang=js 24:11-28
export 'colorVariantClass' (imported as 'colorVariantClass') was not found in '@inkline/inkline/mixins' (possible exports: CollapsibleMixin, FormComponentMixin, LinkableMixin, PopupControlsMixin, PopupMixin, defaultPropValue, sizePropValidator)

export 'Inkline' (imported as 'Inkline') was not found in '@inkline/inkline'

**Comment by alexgrozav** [comment]
Hi @TheDirigible, is there any reason for not using the conventional Vue CLI or Vite.js setup? Looks to me like an issue with ESM imports/exports.

**Reply by TheDirigible** [reply]
Yes, I don't like the Vue CLI or Vite.  They are overcomplex.  Webpack works fine for the build (which is what vue cli uses internally anyway).  This lets me use Vue as a library, instead of as a framework, and have more control over the build.  BTW I have evaluated 20 ui component libs, and this is the only one that doesn't work with webpack.

**Reply by alexgrozav** [reply]
Any chance you can provide a small reproduction repo so I can look into it?

**Reply by TheDirigible** [reply]
Sure, the config is generated so I have extracted a basic repro from it.  Just change the basePath in webpack.config.js.  To make this build successfully, only comment out the import inkline.
[tmp3.zip](https://github.com/inkline/inkline/files/9550230/tmp3.zip)
